Formula/g/gensio.rb

```diff
@@ -0,0 +1,47 @@
+class Gensio < Formula
+  desc "Stream I/O Library"
+  homepage "https://github.com/cminyard/gensio"
+  url "https://downloads.sourceforge.net/project/ser2net/ser2net/gensio-2.7.5.tar.gz"
+  sha256 "2e55ac602062b0807a97cf3d9f0b944bc9667d14b76d67476c579d0615ea574b"
+  license all_of: ["LGPL-2.1-only", "GPL-2.0-only", "Apache-2.0"]
+
+  depends_on "go" => [:build]
+  depends_on "pkg-config" => [:build]
+  depends_on "swig" => [:build]
+  depends_on "glib"
+  depends_on "openssl@3"
+  depends_on "portaudio"
+  depends_on "python@3.11"
+
+  def python3
+    "python3.11"
+  end
+
+  def install
+    # https://rubydoc.brew.sh/Formula.html#std_configure_args-instance_method
+    system "./configure", *std_configure_args.reject { |s| s["--disable-debug"] },
```

Comment:
What does it error with specifically if we don't reject this?

Reply:
It says:
```
configure: WARNING: unrecognized options: --disable-debug
```
and the docs I mentioned at https://docs.brew.sh/Formula-Cookbook#install-the-formula say:
```
Check the top of the e.g. ./configure output. Some configure scripts do not recognise e.g. --disable-debug. If you see a warning about it, remove the option from the formula.
```
There are only a few formulae that do this:
```
g/guile-gnutls.rb:    system "./configure", *std_configure_args.reject { |s| s["--disable-debug"] },
lib/libantlr3c.rb:      system "./configure", *std_configure_args.reject { |s| s["--disable-debug"] },
lib/libpthread-stubs.rb:    system "./configure", *std_configure_args.reject { |s| s["--disable-debug"] }
```
but I'm guessing there are lots of them that are getting the error and ignoring it.

I was really just doing what the docs told me to do.